pkg/app/updatedisc/discovery_updater.go

```diff
@@ -52,7 +52,9 @@ func (u *serviceUpdater) Start() {
 
 	u.wg.Add(1)
```

Comment:
Do we still need the waitgroup?

Reply:
I think goroutine for `updaterloop` was for other reason than loop, but now I remove them.